src/core/tests/frontend/paddle/op_fuzzy.cpp

```diff
@@ -126,6 +126,12 @@ static const std::vector<std::string> models{
     std::string("elementwise_mul4"),
     std::string("elementwise_pow4"),
     std::string("elementwise_sub4"),
+    std::string("elementwise_floordiv_int32_1"),
+    std::string("elementwise_floordiv_int32_2"),
+    std::string("elementwise_floordiv_int32_3"),
+    std::string("elementwise_floordiv_int64_1"),
+    std::string("elementwise_floordiv_int64_2"),
+    std::string("elementwise_floordiv_int64_3"),
```

Comment:
Paddle floor_div support float and integers. Can you test float too?

Reply:
> Paddle floor_div support float and integers. Can you test float too?
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/floor_divide_cn.html#floor-divide  
paddle not support float

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/6076976/192415164-791c31e7-b68d-4f66-a466-b03ee8bb2aa5.png">

Reply:
That's fine. Then leave float there. Thanks.